src/components/ui/sign-description.js

```diff
@@ -0,0 +1,75 @@
+const SignDescription = () => (
+  <div className="SignDescription">
+    <div className="SignDescription__img">
+      <img src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/p/5/005/084/138/2e66790.jpg" />
+    </div>
+    <span>We managed to build, launch and succeed with our Pokemon Go map app in just 48 hours of programming.
+    <span className="SignDescription__author">Artur Czemiel<span>developer & founder, Aexol</span></span>
+    </span>
+
+    <style jsx>{`
+      .SignDescription__img {
+        width: 48px;
+        height: 48px;
+        border-radius: 24px;
+        overflow: hidden;
+        margin-bottom: 24px;
+        margin-top: 20px;
+      }
+
+      .SignDescription__img img{
+        width: 48px;
+        height: 48px;
+      }
+
+      span{
+        display: block;
+        margin: 0;
+        width: 205px;
+        height: 130px;
+        font-family: Roboto;
```

Comment:
Roboto is default font. Why did you put it here? Don't just copy paste all css properties from zeplin. 

Also margin, width, height - remove those. This element should fill parent.